.github/workflows/pre-release-publish-on-tag.yaml

```diff
@@ -115,13 +115,16 @@ jobs:
           # pick anything before the first dash as a version number
           VERSION=$(sed -E 's/^([^-]+)-.*/\1/' <<< $TAG)
 
+          # Install this dev package to gather changelog entries
+          composer global require automattic/jetpack-changelogger
```

Comment:
Correct me if I'm wrong, but is `global` used here to avoid `jetpack-changelogger` to be included into the release package?

Reply:
@dmvrtx - yes, that's my intention.

Reply:
Anyway, I've specified the version (commit: a6b295b513b77455129a2e0b0a841c47512795d8) to make it compatible with composer.json:

https://github.com/Automattic/woocommerce-payments/blob/a6b295b513b77455129a2e0b0a841c47512795d8/composer.json#L45-L45

Reply:
Thanks! It was not clear from PR description.